src/bloqade/analog/task/base.py

```diff
@@ -115,10 +115,6 @@ def from_compile_results(
     @abc.abstractmethod
     def geometry(self) -> Geometry: ...
 
-    @property
-    @abc.abstractmethod
-    def parallel_decoder(self) -> ParallelDecoder: ...
-
```

Comment:
This shouldn't be deleted, this is needed to get things working with the Report object.